ironfish/src/node.ts

```diff
@@ -282,7 +288,7 @@ export class IronfishNode {
   }
 
   onPeerNetworkReady(): void {
-    this.captain.onPeerNetworkReady()
+    void this.syncer.start()
 
     // this.captain.blockSyncer.treesSynced &&
```

Comment:
I think this could be deleted too if you want to delete the commented code

Reply:
Fixed in another PR